tools/runk/src/main.rs

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright 2021 Sony Group Corporation
+//
+
+use runk::container::ContainerAction;
+use runk::create::create_command;
+use runk::delete::delete_command;
+use runk::kill::kill_command;
+use runk::run::run_command;
+use runk::spec::spec_command;
+use runk::start::start_command;
+use runk::state::state_command;
+
+use anyhow::{bail, Result};
+use clap::{crate_name, App, AppSettings, Arg, SubCommand};
+use slog::o;
+use std::fs::OpenOptions;
+use std::path::Path;
+
+const CONFIG_FILE: &str = "config.json";
+const DEFAULT_STATE_ROOT: &str = "/run/runk";
+const DEFAULT_LOG_LEVEL: slog::Level = slog::Level::Info;
+
+async fn real_main() -> Result<(), anyhow::Error> {
+    let id = Arg::with_name("id")
+        .required(true)
+        .takes_value(true)
+        .help("unique identifier of the container");
+
+    let bundle = Arg::with_name("bundle")
+        .long("bundle")
+        .short("b")
+        .default_value(".")
+        .required(true)
+        .help("path to the root of the bundle directory");
+
+    let pid_file = Arg::with_name("pid-file")
+        .long("pid-file")
+        .takes_value(true)
+        .help("specify the file to write the process id to");
+
+    let console_socket = Arg::with_name("console-socket")
+        .long("console-socket")
+        .takes_value(true)
+        .help("path to an AF_UNIX socket which will receive a file descriptor referencing the master end of the console's pseudoterminal");
+
+    let no_pivot = Arg::with_name("no-pivot")
+        .long("no-pivot")
+        .help("do not use pivot root to jail process inside rootfs");
+
+    let matches = App::new("runk")
+        .version("0.1.0")
+        .author("The Kata Containers community <kata-dev@lists.katacontainers.io>")
```

Comment:
Can you use clap's `crate_name!()`, `crate_version!()` and `crate_authors!()` macros here instead of hard-coding the values?

Reply:
I fixed it.